health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/index.js

```
@@ -150,32 +154,33 @@ const App = ({ path, stateCode, userType, tenants,BOUNDARY_HIERARCHY_TYPE, hiera
-         <PrivateRoute path={`${path}/setup-microplan`} component={() => <SetupMicroplan hierarchyType={BOUNDARY_HIERARCHY_TYPE} hierarchyData={hierarchyData} />} />
-         <PrivateRoute path={`${path}/microplan-search`} component={() => <MicroplanSearch></MicroplanSearch>} /> 
-         <PrivateRoute path={`${path}/user-management`} component={() => <UserManagement></UserManagement>} />
-         <PrivateRoute path={`${path}/user-download`} component={() => <UserDownload/>} />
-
-         <PrivateRoute path={`${path}/campaign-boundary`} component={() => <CampaignBoundary/>} /> 
-         <PrivateRoute path={`${path}/test`} component={() => <SearchUnderJurisdiction></SearchUnderJurisdiction>} /> 
-         <PrivateRoute path={`${path}/table`} component={() => <TableNew />} />
-         <PrivateRoute path={`${path}/pop-inbox`} component={() => <PopInbox />} />
-         <PrivateRoute path={`${path}/upload-user`} component={() => <UserUpload/>} /> 
+        <PrivateRoute path={`${path}/setup-microplan`} component={() => <SetupMicroplan hierarchyType={BOUNDARY_HIERARCHY_TYPE} hierarchyData={hierarchyData} />} />
+        <PrivateRoute path={`${path}/microplan-search`} component={() => <MicroplanSearch></MicroplanSearch>} />
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)